cmd/rpcdaemon/commands/eth_callMany.go

```diff
@@ -217,6 +217,9 @@ func (api *APIImpl) CallMany(ctx context.Context, bundles []Bundle, simulateCont
 		if err != nil {
 			return nil, err
 		}
+
+		_ = st.FinalizeTx(rules, state.NewNoopWriter())
```

Comment:
can you please check, if `SoftFinalise` also fixing your problem?

Reply:
Seems like it doesn't

Reply:
Let me know if any more changes are required :)